Makefile

```diff
@@ -21,6 +21,8 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+LD_FLAGS += -X github.com/kubeshop/kusk-gateway/pkg/analytics.kuskGAMeasurementID=$(ANALYTICS_TRACKING_ID) -X github.com/kubeshop/testkube/pkg/analytics.kuskGAApiSecret=$(ANALYTICS_API_KEY)
```

Comment:
`-X github.com/kubeshop/testkube/pkg/analytics.kuskGAApiSecret=$(ANALYTICS_API_KEY)`?

Reply:
Isn't it supposed to be kusk and not testkube?